words.html

```diff
@@ -901,69 +904,68 @@
+ale potrzebna jest też wiedza na temat możliwych przyszłych interakcji <i>zanim</i> pojawi sie zaufanie.
 </p>
 <p id="conclusion_2_a">
-2. POSSIBLE WIN-WINS
+2. MOŻLIWA WYGRANA OBU STRON
 </p>
 <p id="conclusion_2_a2">
-You must be playing a non-zero-sum game,
-a game where it's at least possible that <i>both</i> players can be better off -- a win-win.
+Musisz grać w grę o sumie niezerowej — to gra, w której możliwe jest,<br>
+że <i>obydwaj</i> gracze będą zadowoleni.
 </p>
 <p id="conclusion_3_a">
-3. LOW MISCOMMUNICATION
+3. MAŁO NIEPOROZUMIEŃ
 </p>
 <p id="conclusion_3_a2">
-If the level of miscommunication is <i>too</i> high, trust breaks down.
-But when there's a little bit of miscommunication, it pays to be <i>more</i> forgiving.
+Jeśli nieporozumień jest <i>zbyt</i> wiele, zaufanie zaczyna znikać.
+Jednak gdy nieporozumień jest niewiele, opłaca się być <i>bardziej</i> wyrozumiałym.
 </p>
 <p id="conclusion_4">
-Of course, real-world trust is affected by much more than this.
-There's reputation, shared values, contracts, cultural markers, blah blah blah.
-And let's not forget...
+Oczywiście w prawdziwym świecie na zaufanie ma wpływ znacznie więcej czynników.
+Jest reputacja, wspólne wartości, umowy, markery kulturowe i tak dalej...
+Nie zapominajmy też o...
 </p>
 <p id="conclusion_btn">
-...the <i>biggest</i> lesson. &rarr;
+...<i>najważniejszej</i> lekcji. &rarr;
 </p>
 
 <p id="outro_1">
-If there's one big takeaway<br>
-from <i>all</i> of game theory, it's this:
+Jeśli jest jedna rzecz, którą powinieneś<br>
+wynieść z <i>całej</i> teorii gier, jest to:
 <br><br>
-<b>What the game is, defines what the players do.</b><br>
-Our problem today isn't just that people are losing trust,<br>
-it's that our <i>environment</i> acts against the evolution of trust.
+<b>Zasady gry definiują to, co robią gracze.</b><br>
+Współczesnym problemem jest nie tylko to, że ludzie tracą zaufanie<br>
+ale również <i>środowisko</i> działające na niekorzyść zaufania.
 <br><br>
-That may seem cynical or naive -- that we're "merely" products of our environment --
-but as game theory reminds us, we <i>are</i> each others' environment.
-<b>In the short run, the game defines the players. But in the long run,
-it's us players who define the game.</b>
+Może wydawać się cyniczne lub naiwne, że jesteśmy "jednie" produktem naszego środowiska —
+ale teoria gier przypomina nam, że środowiskiem innych jesteśmy <i>my sami</i>.<br>
+<b>Na krótką metę, to gra definiuje graczy.<br>
+Ale tak naprawdę to my, gracze, definiujemy grę.</b>
 <br><br>
-So, do what <i>you</i> can do, to create the conditions necessary to evolve trust.
-Build relationships. Find win-wins. Communicate clearly.
-Maybe then, we can stop firing at each other, get out of our own trenches,
-cross No Man's Land to come together...
+Co <i>Ty</i> możesz zrobić, aby stworzyć warunki potrzebne do budowy zaufania?
+Buduj relacje, szukaj wspólnych korzyści, komunikuj sie jasno.
```

Comment:
!TODO!

„Twórz relacje” jest powtórzeniem po „stworzyć warunki”, choć dzieli je trochę więcej wyrazów niż alternatywa z „budować” i trudniej zauważyć to powtórzenie.